components/desktop/KnitMenu.vue

```diff
@@ -0,0 +1,223 @@
+<template>
+  <nav class="knitMenu" :class="{knitHighMenu: isHighMenu, knitShortMenu: !isHighMenu}">
+    <div class="knitLogotypeWrapper">
+      <img class="knitLogotype" src="knit-logotype.svg" alt="logo KNIT">
```

Comment:
A website logo in every website should be wrapped to `h1` and link to the main page. An `alt` attribute of this image should be KNIT or KNIT homepage, without 'logo' part.